Samples/Common/Sources/LibraryBridge/RecordingSample.swift

```diff
@@ -27,9 +27,33 @@
 import Foundation
 import KSCrashRecording
 
-public class RecordingSample {
-    public static func simpleInstall() {
-        let config = KSCrashConfiguration()
-        KSCrash.shared().install(with: config)
+public struct RecordingSample {
+    public enum InstallationError: Error, LocalizedError {
+        case kscrashError(String)
+        case unexpectedError(String)
+
+        public var errorDescription: String? {
```

Comment:
Does this go to `localizedDescription`?

Reply:
This is part of the `LocalizedError` protocol and the system primarily uses it to create `localizedDescription`. Without it, it just prints an error code. With this, I tried not to expose KSCrash structs to the UI layer.